lib/Runtime/Debug/DiagObjectModel.cpp

```diff
@@ -2483,7 +2483,7 @@ namespace Js
                                         else
                                         {
                                             const PropertyRecord* propertyRecord;
-                                            objectContext->GetOrAddPropertyRecord(obj->GetSz(), obj->GetLength(), &propertyRecord);
+                                            objectContext->GetOrAddPropertyRecord(obj, &propertyRecord);
```

Comment:
this is an else part, the 'if' part already checking the VirtualTableInfo thing. That code is not needed anymore right? Could you remove it>

Reply:
good idea. will do separately. (since this PR is merged)